hypervisor/partition/vm_description.c

```diff
@@ -76,43 +86,67 @@ static struct vpci_vdev_array vpci_vdev_array1 = {
 };
 
 static struct vpci_vdev_array vpci_vdev_array2 = {
-	.num_pci_vdev = 2,
+	.num_pci_vdev = 3,
 
 	.vpci_vdev_list = {
 	 {/*vdev 0: hostbridge*/
-	  .vbdf = {.b = 0x00U, .d = 0x00U, .f = 0x00U},
+	  .vbdf = { .b = 0x00U, .d = 0x00U, .f = 0x00U },
```

Comment:
keep the style consistent,  remove space before .b, remove space before }

Reply:
Addressed review comment.